constant/index.ts

```diff
@@ -2,6 +2,12 @@ export const IPFS_VIEW_GATEWAY_URL = 'https://ipfs.io/ipfs'
 
 export const MINIMAL_PRICE = 0.99
 export const DEFAULT_PRICE = 4.99
+export const MAXIMUM_PRICE = 199.99
+
+export const USD_PRICING_OPTIONS = [0, ...Array.from(
+  { length: (MAXIMUM_PRICE - MINIMAL_PRICE) + 1 },
+  (_, i) => MINIMAL_PRICE + i
+)]
```

Comment:
No need to run `map()` every time
```suggestion
)].map(option => ({
  label: option.toFixed(2),
  value: option
}))
```